bot/monkey_patches.py

```diff
@@ -50,3 +51,26 @@ async def honeybadger_type(self, channel_id: int) -> None:  # noqa: ANN001
             pass
 
     http.HTTPClient.send_typing = honeybadger_type
+
+
+class FixedPartialMessageConverter(commands.PartialMessageConverter):
+    """
+    Make the Message converter infer channelID from the given context if only a messageID is given.
```

Comment:
Yea, I guess it's technically both, since the MessageConverter just uses util functions from the PartialMessageConverter.

I'm going to say that this word message refers to the generic message and covers both, don't think it's something that needs changing :D